modules/nw-cluster-mtu-change.adoc

```diff
@@ -45,6 +46,8 @@ endif::local-zone,wavelength-zone,post-aws-zones,outposts[]
 * You have installed the {oc-first}.
 * You have access to the cluster using an account with `cluster-admin` permissions.
 * You have identified the target MTU for your cluster. The MTU for the OVN-Kubernetes network plugin must be set to `100` less than the lowest hardware MTU value in your cluster.
+* If your nodes are physical machines, ensure that the cluster network, which includes network switches, supports jumbo frames. 
+* If your nodes are virtual machines (VMs), ensure that the hypervisor and the connected network switches support jumbo frames.
```

Comment:
Does it change the meaning too much to have parallel structure here?
```suggestion
* If your nodes are physical machines, ensure that the cluster network and the connected network switches support jumbo frames. 
* If your nodes are virtual machines (VMs), ensure that the hypervisor and the connected network switches support jumbo frames.
```
I find the first sentence a bit easier to understand this way. If it changes the meaning too much, please feel free to disregard.

Reply:
:heavy_check_mark: